src/app/tests/suites/certification/Test_TC_CC_10_1.yaml

```diff
@@ -0,0 +1,1041 @@
+                value: G1
+              - name: "SceneID"
+                value: 0x01
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: G1
+              - name: "SceneID"
+                value: 0x01
+              - name: "TransitionTime"
+                value: 0
+              - name: "ExtensionFieldSets"
+                constraints:
+                    type: list
+                    contains:
+                        [
+                            {
+                                ClusterID: 0x0300,
+                                AttributeValueList:
+                                    [
+                                        {
+                                            AttributeID: 0x0003,
+                                            ValueUnsigned16: 32768,
+                                        },
+                                        {
+                                            AttributeID: 0x0004,
+                                            ValueUnsigned16: 19660,
+                                        },
+                                        {
+                                            AttributeID: 0x4000,
+                                            ValueUnsigned16: 20000,
+                                        },
+                                        {
+                                            AttributeID: 0x0001,
+                                            ValueUnsigned8: 50,
+                                        },
+                                        {
+                                            AttributeID: 0x4002,
+                                            ValueUnsigned8: 0,
+                                        },
+                                        {
+                                            AttributeID: 0x4003,
+                                            ValueUnsigned8: 0x00,
+                                        },
+                                        {
+                                            AttributeID: 0x4004,
+                                            ValueUnsigned16: 25,
+                                        },
+                                        {
+                                            AttributeID: 0x0007,
+                                            ValueUnsigned16: 48633,
+                                        },
+                                        {
+                                            AttributeID: 0x4001,
+                                            ValueUnsigned8: 0x03,
+                                        },
+                                    ],
+                            },
+                        ]
+
+      # Note : There are too many possible combinations of the extension field sets depending on the supported features, so if they are not all supported, this step will be manual
```

Comment:
not going to block FC on this, but this is fairly key to the test, and it's manual, which means it's error prone. can you assess parts of the extension field sets per feature? ie, is this a separable problem?

Reply:
Initially tried doing this per feature using the `contains` check with yaml, but when specifying contain for a list of extension field set, it expects the exact specific set to be inside the list, i.e.

```
                   contains:
                        [
                            { ClusterID: 0x0300, AttributeValueList: [ { AttributeID: 0x0003, ValueUnsigned16: 32768, },{ AttributeID: 0x0004,ValueUnsigned16: 19660, },],},
                        ]
```
Would look explicitely for an extensionfield set containing only:
                            `{ ClusterID: 0x0300, AttributeValueList: [ { AttributeID: 0x0003, ValueUnsigned16: 32768, },{ AttributeID: 0x0004,ValueUnsigned16: 19660, },],},`
                            And fail if it has other fields.

Reply:
The test is now converted to python and fully automated